src/MBrace.Azure/paket.template

```diff
@@ -36,15 +36,6 @@ files
 	../../bin/Microsoft.ServiceBus.dll ==> tools
 	../../bin/Microsoft.WindowsAzure.Configuration.dll ==> tools
 	../../bin/Microsoft.WindowsAzure.Storage.dll ==> tools
-	../../bin/Hyak.Common.dll ==> tools
```

Comment:
I looked at MBrace.Azure.Management paket.template and didn't see these lines.  I suppose the idea that there isn't a "tools" directory for MBrace.Azure.Management?